Main.cpp

```diff
@@ -32,9 +32,12 @@ int appLauncherMain(int argc, char** argv)
 
   ctkAppArguments appArguments(argc, argv);
 
-  // See http://qt-project.org/doc/qt-4.8/qapplication.html#QApplication
   appArguments.setArgumentToFilterList(
         ctkAppArguments::ArgToFilterListType()
+
+        //
+        // Qt4 built-in arguments - See https://qt-project.org/doc/qt-4.8/qapplication.html#QApplication
```

Comment:
Can you describe in a comment why filtering is needed?

What is the impact of filtering? For example if a CLI module is started with the launcher and has a `-reverse` argument then is filtering going to:
- A. prevent the launcher (as a Qt application) from applying `-reverse` for itself, or
- B. `-reverse` argument is not passed on to the launched CLI module, or
- C. something else?